### PR TITLE
What: Update Joint-review template

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* Need to provide information regarding review participants within the template

This change addresses the need by:

* Breaking Metadata into two sections one for the review, the other for the project.

### CODE REVIEW

**Comment by TheFoxAtWork** [comment]
CC @lumjjb

File: assessments/guide/joint-review.md [hunk 1]
```diff
@@ -43,31 +45,44 @@ from their repo into the self-assessment.
...
 |   |  |
 | -- | -- |
 | Software | A link to the software’s repository.  |
 | Security Provider | Yes or No. Is the primary function of the project to support the
-security of an integrating system?  |
-| Languages | languages the project is written in |
-| SBOM | Software bill of materials.  Link to the libraries, packages, versions used by
- the project, may also included
-irect dependencies. |
+security of an integrating system?  | | Languages | languages the project is
```
Comment (lumjjb) [inline assessments/guide/joint-review.md:67]: lol so actually the original file doesn't work, can't span tables over multiple lines, would need to be on the same line, the linter should be configured already to handle exceptions for this https://github.com/cncf/tag-security/blob/main/ci/lint-config.json#L5

File: assessments/guide/joint-review.md [hunk 2]
```diff
@@ -85,35 +100,38 @@ is meant to provide (e.g., Flibble only allows parties with an authorization key
...
 
 ## Intended Use
 
-* Target Users and Use Cases. Provide a mapping from [standard personas](../../usecase-personas)
-to the nomenclature used in your project docs (which you should then use
-consistently for the remainder of this document).  Describe the scenarios in
-which the project is expected to be used. This must be specific enough to
-provide context for analysis. For example:
+* Target Users and Use Cases. Provide a mapping from [standard
+  personas](../../usecase-personas) to the nomenclature used in your project
```
Comment (lumjjb) [inline assessments/guide/joint-review.md:127]: i know this isn't in scope for the review, but would be cleaner
```suggestion
  personas](/usecase-personas) to the nomenclature used in your project
```

File: ci/spelling-config.json [hunk 3]
```diff
@@ -28,6 +28,11 @@
         "CNSWP",
         "CNSWP's",
         "SLSA",
-        "CNSMAP"
+        "CNSMAP",
+        "SBOM",
+        "Flibble",
+        "minimalistic",
+        "usecase",
```
Comment (lumjjb) [inline ci/spelling-config.json:35]: hmm this isn't used in the current patch? Is it needed?
```suggestion
```

**Review by lumjjb — COMMENTED** [review]
couple nits/comments

File: ci/spelling-config.json (hunk 3 above)
Comment (TheFoxAtWork) [reply ci/spelling-config.json]: >CSpell: Files checked: 1, Issues found: 1 in 1 files
/usr/src/app/assessments/guide/joint-review.md:125:14 - Unknown word (usecase)

**Review by anvega — APPROVED** [review]

**Pull request closed without merging**